### PR TITLE
Add missing langchain dependencies

### PR DESCRIPTION
**Pull request by suhaibmujahid**
We started using these dependencies in https://github.com/mozilla/bugbug/pull/4133

### CODE REVIEW

File: bugbug/generative_model_tool.py [hunk 1]
```diff
@@ -6,7 +6,7 @@
 from abc import ABC, abstractmethod
 from typing import Any
 
-from langchain.llms.human import HumanInputLLM
+from langchain_community.llms import HumanInputLLM
```
Comment (benjaminmah) [inline bugbug/generative_model_tool.py:9]: According to the LangChain docs, `HumanInputLLM` may need to be imported from `langchain_community.llms.human`.

See: [LangChain HumanInputLLM](https://api.python.langchain.com/en/latest/llms/langchain_community.llms.human.HumanInputLLM.html#langchain_community.llms.human.HumanInputLLM)
```suggestion
from langchain_community.llms.human import HumanInputLLM
```

**Review by benjaminmah — COMMENTED** [review]
Thank you @suhaibmujahid! I've only suggested one change but the rest LGTM!

File: bugbug/generative_model_tool.py (hunk 1 above)
Comment (suhaibmujahid) [reply bugbug/generative_model_tool.py:9]: I notice the need for the update through the following message:
```
/lib/python3.10/site-packages/langchain/_api/module_import.py:92: LangChainDeprecationWarning: Importing HumanInputLLM from langchain.llms is deprecated. Please replace deprecated imports:

>> from langchain.llms import HumanInputLLM

with new imports of:

>> from langchain_community.llms import HumanInputLLM
You can use the langchain cli to **automatically** upgrade many imports. Please see documentation here https://python.langchain.com/v0.2/docs/versions/v0_2/ 
```

And I applied the change using `langchain-cli `:
```
langchain-cli migrate  bugbug/generative_model_tool.py
```


It seems that the class is re-exported from `langchain_community.llms`: https://github.com/langchain-ai/langchain/blob/ee32369265103ac64b412a73a758ae0d1649cafe/libs/community/langchain_community/llms/__init__.py#L281-L284 

Here is an example usage: https://github.com/langchain-ai/langchain/blob/dcec133b85381a9e40ad8afa2c6f8226b8db1e3b/libs/langchain/langchain/llms/human.py#L6

**Review by benjaminmah — APPROVED** [review]
LGTM

**Pull request merged**